app/views/layouts/navigation.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-xs-12 col-sm-1 no-horiz-padding">
+      <ul class="nav nav-pills nav-stacked">
+        <% HqController::NAVIGATION_BUTTONS.each do | _, btn | %>
+          <li role="presentation" <%= (request.path=~ btn[:path_regex]) ? 'class=active' : 'class=dormant' %>>
```

Comment:
I don't like the idea of a path regex for active classes when we have fails routing functionality we could tap into. If we do this our routing then has to be maintained in the routes file and in the `NAVIGATION_BUTTONS` hash